package.json

```diff
@@ -198,6 +198,7 @@
     "nwsapi": "^2.2.1",
     "path-to-regexp": "~8.0.0",
     "patternfly": "~3.59.5",
-    "terser": "~4.8.1"
+    "terser": "~4.8.1",
+    "form-data": "~4.0.4"
```

Comment:
@elsamaryv one small fix, can you please put this in alphabetical order.

Reply:
Also, I believe form-data 2.5.5 also contains the fix. Can we upgrade to that version instead to prevent such a big version jump

Reply:
Yeah @elsamaryv can you first try to upgrade the jest and jest-cli packages then see if it upgrades the form data package. If you are able to upgrade jest and jest-cli see what the most recent version you can get to for those without anything breaking and then see if that causes form-data to upgrade also.